### DISCUSSION TITLE
What is the memory bandwidth of NVIDIA P40

### DISCUSSION THREAD

**Discussion by bobqianic** [ggml-org/llama.cpp#4221] [Q&A] [answered]
Regarding the memory bandwidth of the NVIDIA P40, I have seen two different statements. One is from the NVIDIA official spec, which says `347 GB/s`, and the other is from the TechpowerUP database, which says `694.3 GB/s`. So, what exactly is the bandwidth of the P40? Does anyone know?

<img width="215" alt="image" src="https://github.com/ggerganov/llama.cpp/assets/129547291/48a99bea-45bd-4529-95a6-1a074e82ca1e">
<img width="421" alt="image" src="https://github.com/ggerganov/llama.cpp/assets/129547291/c2aac2f9-548a-4f1f-abc3-6a543ed65ff4">

https://resources.nvidia.com/en-us-virtualization-and-gpus/p40-product-brief

https://www.techpowerup.com/gpu-specs/tesla-p40.c2878

**Comment by sorasoras** [comment, marked as answer]
http://gpubandwidthcalculator.totalh.net/?i=1
346GB/s are probably correct. P40 use GDDR5.

**Comment by bobqianic** [comment]
Yesterday, I sent an email to TechPowerUP to report the error, and today they have already fixed it.
![image](https://github.com/ggerganov/llama.cpp/assets/129547291/0ee40268-0539-4f26-9745-0ffe6f8154fa)
